### ISSUE TITLE
graph_models screenshots

### ISSUE THREAD

**Issue by abitrolly** [django-extensions/django-extensions#1608]
On the https://django-extensions.readthedocs.io/en/latest/graph_models.html page it is quite hard to figure out how results look like for different examples.

**Comment by morenoh149** [comment]
Agreed. What datamodel should we run the graphs on to illustrate the different options?

**Comment by abitrolly** [comment]
I dk. Some of these? https://dev.to/jackdlinke/modern-django-project-examples-58mm
